extensions/amp-render/1.0/component.js

```diff
@@ -110,8 +109,19 @@ export function RenderWithRef(
   const isHtml =
     rendered && typeof rendered == 'object' && '__html' in rendered;
 
+  const refFn = useCallback(
+    (node) => {
+      if (!node?.firstElementChild || !rendered) {
+        return;
+      }
+      onReady?.();
+    },
+    [rendered, onReady]
+  );
+
```

Comment:
Will this change still be necessary if the layout value checks are moved to `isLayoutSupported` in the AMP layer?